qbft/spectest/tests/msg_processing_spectest.go

```diff
@@ -76,8 +78,8 @@ func (test *MsgProcessingSpecTest) Run(t *testing.T) {
 	}
 
 	// test root
-	if test.PostRoot != hex.EncodeToString(postRoot) {
-		diff := comparable3.PrintDiff(test.Pre.State, comparable2.RootRegister[test.PostRoot])
+	if test.PostRoot != hex.EncodeToString(postRoot[:]) {
+		diff := typescomparable.PrintDiff(test.Pre.State, test.PostState)
```

Comment:
There is a case that PostState may be nil (for tests we didn't do SC, or for tests at `ssv` that run via json

Reply:
in case postState is null, the printDiff will output null for Target

Reply:
I think it may be a bit ugly when running the spec tests against `ssv` but it is not the worst thing.
You have plenty of work so you can choose to either change or not.

Reply:
opened this issue for running against `ssv` with no diffs :
https://github.com/bloxapp/ssv-spec/issues/213

From looking at the code I think the issue also existed before. Because I didn't that we serialize the `sc` files